Spring-JMS/src/main/java/com/ibm/mq/samples/jms/spring/level114/MQConfiguration114.java

```diff
@@ -41,6 +41,12 @@ public class MQConfiguration114 {
     @Bean
     public MQConnectionFactory mqConnectionFactory() throws JMSException {
         MQConfigurationProperties properties = new MQConfigurationProperties();
+        // Uncomment when running the 114 sample
```

Comment:
Configuration should not be hardcoded. Remove these lines, even if they are only comments.